app/controllers/api_controller/blueprints.rb

```diff
@@ -0,0 +1,28 @@
+class ApiController
+  module Blueprints
+    def create_resource_blueprints(_type, _id, data)
+      attributes = data.except("bundle")
```

Comment:
It looks like this is already being handled elsewhere. I've added some tests to ensure that we don't regress on that behavior